.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```

Comment:
In the .gitignore of our repository, please put `.idea/` so that your IDEs configuration files aren't included in the repository. Thank you!

You're also going to need to delete these files from the repository by doing as follows:
`git rm -r --cached .idea/`
NOTE: the `--cached` option makes it so that the files are not deleted locally.